packages/shortcuts-extension/style/base.css

```
@@ -0,0 +1,394 @@
+}
+
+.jp-Shortcuts-ShortcutKeysContainer {
+  font-size: var(--jp-code-font-size);
+  font-family: var(--jp-ui-font-family);
+  display: flex;
+}
+
+.jp-Shortcuts-ConflictContainer {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 6px 12px;
+  margin-left: 20%;
+}
+
+.jp-Shortcuts-ShortcutKeys {
+  border-width: var(--jp-border-width);
+  border-color: var(--jp-layout-color0);
+  border-radius: var(--jp-border-radius);
+  padding: 5px 6px;
+  margin: 3px 0;
+}
+
+.jp-Shortcuts-Or {
+  margin-right: 12px;
+  margin-left: 12px;
+  margin-top: 8px;
+  color: var(--jp-content-font-color3);
+  display: none;
+}
+
+.jp-Shortcuts-Or:hover {
+  display: block;
+}
+
+.jp-Shortcuts-OrTwo {
+  margin-right: 12px;
+  margin-left: 12px;
+  margin-top: 8px;
+  color: var(--jp-content-font-color3);
+  display: block;
+}
+
+.jp-Shortcuts-Comma {
+  margin-top: 10px;
+  margin-right: 2px;
+  margin-left: 2px;
+}
+
+.jp-Shortcuts-Plus {
+  display: none;
+  background: var(--jp-brand-color3);
+  border-color: var(--jp-layout-color0);
+  border-radius: var(--jp-border-radius);
+  border-width: var(--jp-border-width);
+  margin: 3px 0;
+  padding: 5px 6px;
+}
+
+.jp-Shortcuts-Plus:hover {
+  background-color: var(--jp-brand-color2);
+}
+
+.jp-Shortcuts-Plus:active {
+  background-color: var(--jp-brand-color2);
+}
+
+.jp-Shortcuts-Reset {
+  color: var(--jp-brand-color2);
+  padding-left: 10px;
+}
+
+.jp-Shortcuts-Reset:hover {
+  color: var(--jp-brand-color1);
+}
+
+.jp-Shortcuts-SourceCell {
+  display: inline-block;
+}
+
+/* Shortcut List Style */
+.jp-Shortcuts-ShortcutList {
+  width: 100%;
+  display: table;
+  border-collapse: collapse;
+}
+
+.jp-Shortcuts-ShortcutListContainer {
+  overflow-y: scroll;
+  border: var(--jp-border-width) solid var(--jp-border-color1);
+}
+
+/* Shortcut Title Item Style */
+.jp-Shortcuts-Header {
+  display: flex;
+  cursor: pointer;
+}
+
+.jp-Shortcuts-Header:hover .jp-ShortcutTitleItem-sortButton[fill],
+.jp-Shortcuts-Header:focus .jp-ShortcutTitleItem-sortButton[fill] {
+  fill: var(--jp-ui-font-color0);
+}
+
+.jp-Shortcuts-Header:active .jp-ShortcutTitleItem-sortButton {
+  outline: none;
+}
+
+.jp-Shortcuts-CurrentHeader div {
+  color: var(--jp-ui-font-color0);
+  font-weight: bold;
+}
+
+/* Shortcut UI Style */
+
+.jp-Shortcuts-ShortcutUI {
+  display: flex;
+  flex-direction: column;
+  font-size: var(--jp-ui-font-size1);
+  font-family: var(--jp-ui-font-family);
+  color: var(--jp-content-font-color1);
+  min-width: 450px;
+  width: 100%;
+}
+
+/* TopNav Style */
+.jp-Shortcuts-Top {
+  display: block;
+}
+
+.jp-Shortcuts-TopNav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+  font-size: var(--jp-ui-font-size1);
+  background-color: var(--jp-layout-color0);
+}
+
+.jp-Shortcuts-Symbols {
+  padding: 0 12px;
+}
+
+.jp-Shortcuts-Symbols td:nth-child(2) {
+  padding-right: 10px;
+}
+
+.jp-Shortcuts-Search {
+  height: 30px;
+}
+
+.jp-Shortcuts-Search input {
```

This is a generic tag selector, so lets shorten the search to avoid performance penalty.

```suggestion
.jp-Shortcuts-Search > input {
```